azure/packages/azure-service-utils/package.json

```diff
@@ -110,6 +110,12 @@
 		}
 	},
 	"typeValidation": {
-		"broken": {}
+		"disabled": true,
+		"broken": {
```

Comment:
What does disable and broken mean?

Reply:
broken is ignored when disabled is true. It's useful in cases where the typetests themselves are busted (what I think is happening here) and should be disabled but once fixed you don't want to have to hunt around for the expected disables.